src/openforms/submissions/api/viewsets.py

```diff
@@ -119,6 +120,7 @@ def perform_create(self, serializer):
         conf = GlobalConfiguration.get_solo()
         if conf.enable_form_variables:
             prefill_variables(serializer.instance)
+            initialise_variables_unrelated_to_a_step(serializer.instance)
```

Comment:
When resuming a form we will have to refresh the static variables?
Maybe we shouldn't persist static variables to databases?

Reply:
Created issue: https://github.com/open-formulieren/open-forms/issues/1761